cdap-common/src/main/java/co/cask/cdap/common/conf/Constants.java

```diff
@@ -1070,6 +1070,7 @@
    */
   public static final class MessagingSystem {
     public static final String LOCAL_DATA_DIR = "messaging.local.data.dir";
+    public static final String LOCAL_TTL_CLEANUP_FREQUENCY = "messaging.local.data.ttl.cleanup.frequency";
```

Comment:
It's good to have the unit as part of the config key. E.g. `messaging.local.data.cleanup.frequency.secs`